src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -339,6 +339,18 @@ public enum Mixins {
             .setApplyIf(() -> Common.config.fixThaumcraftGolemMarkerLoading).addTargetedMod(TargetedMod.THAUMCRAFT)),
 
     // BOP
+    FIX_QUICKSAND_XRAY(new Builder("Fix Xray through block without collision boundingBox")
+            .addMixinClasses("biomesoplenty.MixinBlockMud")
+            .setApplyIf(
+                    () -> Common.config.fixPerspectiveCamera
+                            && Common.config.fixXrayThroughBlockWithoutCollisionBoundingBox)
+            .addTargetedMod(TargetedMod.BOP)),
+    FIX_QUICKSAND_XRAY2(new Builder("Fix Xray through block without collision boundingBox")
```

Comment:
This shouldn't be in the BoP block. Also, I'd name it just `FIX_QUICKSAND_XRAY` and the above one `FIX_QUICKSAND_XRAY_BOP`.